docs/gen-ai/README.md

```diff
@@ -7,9 +7,24 @@ linkTitle: Generative AI
 **Status**: [Development][DocumentStatus]
 
 > [!Warning]
-> The semantic conventions for GenAI and LLM are currently in development.
-> We encourage instrumentation libraries and telemetry consumers developers to
-> use the conventions in limited non-critical workloads and share the feedback
+>
+> Existing GenAI instrumentations that are using
+> [v1.34.0 of this document](https://github.com/open-telemetry/semantic-conventions/blob/v1.34.0/docs/gen-ai/README.md)
+> (or prior):
+>
+> * SHOULD NOT change the version of the GenAI conventions that they emit by default.
+>   Conventions include, but are not limited to, attributes, metric, span and event names,
+>   span kind and unit of measure.
+> * SHOULD introduce an environment variable `OTEL_SEMCONV_EXPERIMENTAL_OPT_IN`
+>   as a comma-separated list of category-specific values. The list of values
+>   includes:
+>   * `gen-ai-latest` - emit the latest (supported by the instrumentation) GenAI
```

Comment:
wdyt of?
```suggestion
>   * `gen-ai-latest-experimental` - emit the latest (supported by the instrumentation) GenAI
```

Reply:
oh I missed that, thx!

Reply:
WDYT about https://github.com/open-telemetry/semantic-conventions/pull/2046/commits/1e977ce277e860af596fa8f3f503c414282d2a53 ?